repos/testUtils/support/setup.js

```diff
@@ -0,0 +1,5 @@
+const { getBrowserContext } = require('HerkinTasks/utils/wolf/setupTestEnvironment')
```

Comment:
This seems backwards to me.
* The task should `require` from the util, not the other way around
* If `setupTestEnvironment` needs to be re-used, we should move it into the testUtils repo
* Then anything that needs it would `require` from that

Reply:
Updated to move `setupTestEnvironment` into the `testUtils` directory. It shouldn't have been in the tasks at all, as it is used for more than just tasks.

Now everything imports it from `HerkinRepos/testUtils/playwright/setupTestEnvironment`

Reply:
Nice, that makes more sense